pkgs/applications/misc/khard/default.nix

```
@@ -18,7 +18,7 @@ python3.pkgs.buildPythonApplication rec {
     atomicwrites
     configobj
     vobject
-    ruamel_yaml
+    ruamel-yaml
     ruamel_base
```

https://github.com/NixOS/nixpkgs/pull/144407#issuecomment-959834865

```suggestion
    ruamel-base
```

---

Fixing that right now. This can be entirely removed because ruamel-yaml propgogtes it.